gap/ClassicalMaximals.gi

```diff
@@ -827,3 +827,17 @@ function(n, q, classes...)
 
     return maximalSubgroups;
 end);
+
+
+BindGlobal("C5SubgroupsSymplecticGroupGeneric",
+function(n, q)
+    local factorisation, p, e, degreeOfExtension, result;
+    
+    factorisation := PrimePowersInt(q);
+    p := factorisation[1];
+    e := factorisation[2];
+
+    # For each prime divisor of e, there is exactly one of these subgroups,
+    # so this is sufficient.
+    return List(PrimeDivisors(e), b -> SubfieldSp(n, p, e, QuoInt(e ,b)));
```

Comment:
You have to remember conjugating the subgroup you constructed by an element of GSp \ Sp in order to get all the conjugacy classes in Sp, see for example ll. 833-834 at https://github.com/hulpke/maxtrans/blob/master/magma/classmax.m